receiver/prometheusoperatorreceiver/README.md

```diff
@@ -0,0 +1,80 @@
+# Simple Prometheus Receiver
+
+The `prometheus_operator` receiver is a wrapper around the [prometheus
+receiver](../prometheusreceiver).
+This receiver allows using [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) 
+CRDs like `ServiceMonitor` and `PodMonitor` to configure metric collection.
+
+Supported pipeline types: metrics
+
+> :construction: This receiver is in **development**, and is therefore not usable at the moment.
+
+## When to use this receiver?
+Choose this receiver, if you use [ServiceMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/main/example/user-guides/getting-started/example-app-service-monitor.yaml) 
+or [PodMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/main/example/user-guides/getting-started/example-app-pod-monitor.yaml) 
+custom resources in your Kubernetes cluster. These can be provided by applications (e.g. helm charts) or manually 
+deployed by users. 
+
+In every other case other Prometheus receivers should be used. 
+Below you can find a short description of the available options.
+
+### Prometheus scrape annotations
+If you use an annotations based scrape configs like `prometheus.io/scrape = true`, then you can use the
+[receivercreator](../receivercreator/README.md). This receiver is able to create multiple receivers based on a set of
+rules, which are defined by the user.
+
+A guide how to use this meta receiver with Prometheus annotations you can find in the
+[examples of the receivercreator](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/receivercreator#examples)
+
+### Static targets
+If a simple static endpoint in or outside the cluster should be scraped, use the [simpleprometheusreceiver](../simpleprometheusreceiver/README.md).
+It provides a simplified interface around the `prometheusreceiver`. Use cases could be the federation of Prometheus
+instances or scraping of targets outside dynamic setups. 
+
+### Prometheus service discovery and manual configuration
```

Comment:
Just documenting some investigation i've done in the past:  Why not just implement PodMonitor and ServiceMonitor using the prometheus service discovery?  That would have the benefit of not needing to shutdown and restart the prometheus receiver when a PodMonitor or ServiceMonitor is modified.

Answer: The prometheus service discovery interface only supports adding new targets, but doesn't support manipulating metrics after they are scraped.  So we wouldn't be able to support metricRelabelConfigs with that approach.